fe/fe-core/src/main/java/com/starrocks/external/iceberg/cost/IcebergTableStatisticCalculator.java

```diff
@@ -0,0 +1,240 @@
+                            nonPartitionPrimitiveColumns,
+                            dataFile.partition(),
+                            dataFile.recordCount(),
+                            dataFile.fileSizeInBytes(),
+                            IcebergFileStats.toMap(idToTypeMapping, dataFile.lowerBounds()),
+                            IcebergFileStats.toMap(idToTypeMapping, dataFile.upperBounds()),
+                            dataFile.nullValueCounts(),
+                            dataFile.columnSizes());
+                } else {
+                    icebergFileStats.incrementFileCount();
+                    icebergFileStats.incrementRecordCount(dataFile.recordCount());
+                    icebergFileStats.incrementSize(dataFile.fileSizeInBytes());
+                    updateSummaryMin(icebergFileStats, partitionFields, IcebergFileStats.toMap(idToTypeMapping,
+                            dataFile.lowerBounds()), dataFile.nullValueCounts(), dataFile.recordCount());
+                    updateSummaryMax(icebergFileStats, partitionFields, IcebergFileStats.toMap(idToTypeMapping,
+                            dataFile.upperBounds()), dataFile.nullValueCounts(), dataFile.recordCount());
+                    icebergFileStats.updateNullCount(dataFile.nullValueCounts());
+                    updateColumnSizes(icebergFileStats, dataFile.columnSizes());
+                }
+            }
+        } catch (IOException e) {
+            throw new UncheckedIOException(e);
+        }
+
+        if (icebergFileStats == null) {
+            return Statistics.builder().build();
+        }
+
+        Statistics.Builder statisticsBuilder = Statistics.builder();
+        double recordCount = icebergFileStats.getRecordCount();
+        for (Map.Entry<Integer, String> idColumn : idToColumnNames.entrySet()) {
+            List<ColumnRefOperator> columnList = colRefToColumnMetaMap.keySet().stream().filter(
+                    key -> key.getName().equals(idColumn.getValue())).collect(Collectors.toList());
+            if (columnList == null || columnList.size() != 1) {
+                LOG.debug("This column is not required column name " + idColumn.getValue() + " column list size "
+                        + (columnList == null ? "null" : columnList.size()));
+                continue;
+            }
+
+            int fieldId = idColumn.getKey();
+            ColumnStatistic.Builder columnBuilder = ColumnStatistic.builder();
+
+            // nulls fraction
+            Long nullCount = icebergFileStats.getNullCounts().get(fieldId);
+            if (nullCount != null) {
+                columnBuilder.setNullsFraction(nullCount / recordCount);
+            } else {
+                columnBuilder.setNullsFraction(-1L / recordCount);
+            }
+
+            // avg row size
+            if (icebergFileStats.getColumnSizes() != null) {
+                Long columnSize = icebergFileStats.getColumnSizes().get(fieldId);
```

Comment:
The column size looks like (size of column * row count), plz add some commment to help easier to understand